spec/01-unit/01-db/01-schema/06-routes_spec.lua

```diff
@@ -1510,4 +1510,43 @@ describe("routes schema (flavor = expressions)", function()
     -- verified by `schema/typedefs.lua`
     assert.truthy(errs["@entity"])
   end)
+
+  it("http route still supports net.port but with warning", function()
```

Comment:
We should also make assertions on the warning messages.

Reply:
Could you give me an example about how to check the warning message? thanks.

Reply:
For integrated tests we use `assert.logfile`; for unit tests we use [`assert.spy`](https://lunarmodules.github.io/busted/).